lib/core/src/main/java/io/atlasmap/actions/StringComplexFieldActions.java

```diff
@@ -72,10 +72,13 @@ public static String concatenate(Concatenate concat, List<String> inputs) {
         }
 
         String delim = concat.getDelimiter() == null ? "" : concat.getDelimiter();
-        StringBuilder builder = new StringBuilder();
+        boolean delimitingEmptyValues = concat.getDelimitingEmptyValues() == null
```

Comment:
Isn't this just 
boolean delimitingEmptyValues = !(concat.getDelimitingEmptyValues() == null)

Reply:
Not sure if I understand your comment. Based on what you've suggested, if getDelimitingEmptyValues returns False, the variable would end up as true.

Reply:
You're right - no ! needed.  Just trying to avoid calling `getDelimitingEmptyValues()` twice.